nondex-core/src/main/java/java/util/concurrent/PriorityBlockingQueue.java

```diff
@@ -752,6 +752,29 @@ public String toString() {
         }
     }
 
+    public String toString() {
```

Comment:
Is there a legitimate reason why you can't call the iterator from here, rather than calling NonDex directly?
I would just change the for loop to for (Object e : iterator()) - or something like that.